packages/web/src/common/store/pages/audio-transactions/sagas.ts

```diff
@@ -153,19 +153,21 @@ function* fetchTransactionMetadata() {
       const { txDetails } = action.payload
       yield* call(waitForLibsInit)
       const libs: AudiusLibs = yield* call(audiusBackendInstance.getAudiusLibs)
-      const response = yield* call(
+      const response = (yield* call(
         [
           libs.identityService!,
           libs.identityService!.getUserBankTransactionMetadata
         ],
         txDetails.signature
-      )
+      )) as any[]
```

Comment:
I must have missed this `any` before! hopefully this helps to fix? https://github.com/AudiusProject/audius-protocol/pull/4731

Reply:
I'd still have to cast though right? Bc type in identity != type in client, we'd have to convert all uses of this type in client to the identity in type to avoid cast unless I'm mistaken?

Reply:
As long as the types are equivalent, it should be ok